packages/maskbook/src/plugins/Snapshot/types.ts

```diff
@@ -0,0 +1,119 @@
+import type snapshot from '@snapshot-labs/snapshot.js'
+
+/**
+ * @param space ENS domain name of space.
+ * Space is a set of proposals by which specific entity can raise for its purpose.
+ * https://docs.snapshot.org/spaces
+ * @param id the identifier of proposal
+ */
+export interface ProposalIdentifier {
+    space: string
+    id: string
+}
```

Comment:
```suggestion
export interface ProposalIdentifier {
    /**
     * ENS domain name of space.
     * Space is a set of proposals by which specific entity can raise for its purpose.
     * https://docs.snapshot.org/spaces
     */
    space: string
    /** the identifier of proposal */
    id: string
}
```

![image](https://user-images.githubusercontent.com/5390719/114984780-a8573c00-9ec4-11eb-92f4-25bc0ce423d8.png)


Otherwise